addons/account_edi_facturx/models/account_edi_format.py

```diff
@@ -186,6 +186,13 @@ def _find_value(xpath, element=tree):
                 vat=_find_value('//ram:' + partner_type + '/ram:SpecifiedTaxRegistration/ram:ID'),
             )
 
+            # Bank Account
+            acc_number = _find_value('//ram:PayeePartyCreditorFinancialAccount/ram:IBANID')
+            if acc_number:
+                bank_account = self._retrieve_bank_account(acc_number, invoice_form.partner_id)
+                if bank_account not in invoice_form.partner_id.bank_ids:
```

Comment:
Actually no, because when it's created it's inactive until the user has explicitly confirmed it. However, I agree this was not clear and the code could be cleaner, so I refactored it a bit.